app/Repository/Member/MemberRepository.php

```diff
@@ -99,6 +99,24 @@ public function getUpdated( int $revisionId ): array {
 		return $this->getMultiple( $revision->getMemberIds() );
 	}
 
+	/**
+	 * @param array $rootGroups
+	 *
+	 * @return array
+	 * @throws ClientException
+	 * @throws GroupNotFoundException
+	 * @throws InvalidFixedValueException
+	 * @throws MemberNotFoundException
+	 * @throws MemberUnknownFieldException
+	 * @throws MultiSelectOverwriteException
+	 * @throws ValueTypeException
+	 * @throws WeblingAPIException
+	 * @throws WeblingFieldMappingConfigException
+	 */
+	public function getAll( array $rootGroups = [] ): array {
+		return $this->find( '', $rootGroups );
```

Comment:
Ist ev. etwas eine Doppelspurigkeit mit Group->getAllMembers(). Macht es Sinn, dass wir 2x fast das gleiche auf unterschiedliche Weise machen?

Reply:
Habe das gerade nochmals genauer angeschaut. In einer idealen Welt würde es definitiv keinen Sinn ergeben. Wenn wir es aber über die Gruppen lösen würden, müssten wir ungecached arbeiten (das `MemberRepository` ist ja überall ungecached) und dann hätten wir ziemlich viele zusätzliche Anfragen, die den Call dann deutlich langsamer machen würden...